libcodechecker/libhandlers/quickcheck.py

```diff
@@ -322,14 +318,14 @@ def __update_if_key_exists(source, target, key):
             output_path=workspace,
             output_format='plist',
             clean=True,
-            jobs=args.jobs,
-            add_compiler_defaults=args.add_compiler_defaults
+            jobs=args.jobs
         )
         # Some arguments don't have default values.
         # We can't set these keys to None because it would result in an error
         # after the call.
         args_to_update = ['skipfile',
```

Comment:
Looks like this list is replicated. In the future, it might be better to refactor it in a way that this list is created together with the arg parsers.